ex3.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pickle
+from datetime import datetime
+import ex1
+import shutil
+
+
+ex1_decoded_files = 'ex1_decoded'
+
+
+def find_file_names_transform(
+
+) -> dict:
+    # 110 (hint from part 2) == 6
+    with open('dirs/ex3/6/6.pkl', 'rb') as f:
+        data: dict = pickle.load(f)
+
+    result = {
+        field_name: {} for field_name in os.listdir('dirs/ex1')
+    }
+
+    for field_name, decoding_data in data.items():
+        field_name: str = field_name
+        decoding_data: dict = decoding_data
```

Comment:
no need to redefine the variable, can just do 
```suggestion
        decoding_data: dict
```
Before the loop would be better